locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/BrooklynImageChooser.java

```diff
@@ -371,7 +371,12 @@ public ImageChooserFromOrdering cloneFor(ConfigBag config) {
             } else {
                 return this;
             }
-        }        
+        }
+
+        @Override
+        public String toString(){
+            return getClass().getName();
+        }
```

Comment:
@aledsage, this tries to avoid the following entry in the catalog  for `JcloudsLocation`.
<img width="962" alt="captura de pantalla 2016-05-31 a la s 10 24 10" src="https://cloud.githubusercontent.com/assets/3704424/15668131/002369d6-271a-11e6-9134-67df64155a9e.png">

I think this impl is enough, maybe [ordering](https://github.com/apache/brooklyn-server/blob/master/locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/BrooklynImageChooser.java#L330)?